include/aws/io/io.h

```diff
@@ -252,6 +252,8 @@ enum aws_io_errors {
 
     AWS_IO_TLS_ERROR_READ_FAILURE,
 
+    AWS_ERROR_PEM_MALFORMED_OBJECT,
```

Comment:
Do you expect other systems to use a different error if the objects themselves are all valid, but the "document" itself is malformed due to missing required objects?

If we fear people will re-use this error-code for bad documents, give it a more generic name like `AWS_ERROR_PEM_MALFORMED`. But if you want distinct error-codes to be created for malformed documents of a specific type, then this is good

Reply:
let me just make it generic AWS_ERROR_PEM_MALFORMED. i dont think its worth distinguishing that its individual objects that are malformed